README.md

```diff
@@ -163,6 +163,9 @@ add_filter( 'pardot_https_regex', 'pardot_custom_filter_https_regex' );
 
 ## Changelog
 
+### 1.5.8
+* Fix - Handled an issue that authentication failure ([Pardot API Error Code 184](https://developer.salesforce.com/docs/marketing/pardot/guide/error-codes.html#numerical-list-of-error-codes)) causes the settings page stuck in loading
```

Comment:
Nit: "causes the settings page to be stuck in a loading state"

Reply:
@pengchensfdc please update the wording here, otherwise, it looks good.